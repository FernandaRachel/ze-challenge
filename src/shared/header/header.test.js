import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';
import 'jest-localstorage-mock';

describe('Header', () => {
    test('snapshot renders', () => {
        const arr = [];
        const component = renderer.create(<Header />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('Expect session storage called', () => {
        const component = renderer.create(<Header />);
        const KEY = 'address',
            VALUE = 'Rua Fake 333';
        dispatch(action.update(KEY, VALUE));
        expect(component.getInstance()).toHaveBeenLastCalledWith(KEY, VALUE);
        const arr = [];
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

