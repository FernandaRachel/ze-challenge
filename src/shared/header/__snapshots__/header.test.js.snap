// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot renders 1`] = `
<div
  className="header"
>
  <img
    className="logo"
    src="ze-logo.png"
  />
  <span>
    
  </span>
</div>
`;
